views/index.slim

```diff
@@ -92,13 +92,12 @@ ul.list
 
 h2#links External Links
 
+h3#Tutorials
```

Comment:
This syntax creates an empty tag `<h3 id="Tutorials"></h3>` with no text. It should probably be `h3 Tutorials` or `h3#tutorials Tutorials`.